### ISSUE TITLE
Support Lavavel 6.1

### ISSUE THREAD

**Issue by mwisner** [glhd/aire#40]
When I try to upgrade to Laravel 6.1 I get:

```  backend git:(feat/vapor-work) ✗ composer upgrade
Loading composer repositories with package information                                                                                                                                                             Updating dependencies (including require-dev)         
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - glhd/aire 1.8.0 requires illuminate/view 5.7.*|5.8.*|6.0.* -> satisfiable by laravel/framework[v6.0.4], illuminate/view[5.7.17, 5.7.18, 5.7.19, v5.7.0, v5.7.1, v5.7.10, v5.7.11, v5.7.15, v5.7.2, v5.7.20, v5.7.21, v5.7.22, v5.7.23, v5.7.26, v5.7.27, v5.7.28, v5.7.3, v5.7.4, v5.7.5, v5.7.6, v5.7.7, v5.7.8, v5.7.9, v5.8.0, v5.8.11, v5.8.12, v5.8.14, v5.8.15, v5.8.17, v5.8.18, v5.8.19, v5.8.2, v5.8.20, v5.8.22, v5.8.24, v5.8.27, v5.8.28, v5.8.29, v5.8.3, v5.8.30, v5.8.31, v5.8.32, v5.8.33, v5.8.34, v5.8.35, v5.8.4, v5.8.8, v5.8.9, v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4].
    - glhd/aire 1.8.0 requires illuminate/events 5.7.*|5.8.*|6.0.* -> satisfiable by laravel/framework[v6.0.4], illuminate/events[5.7.17, 5.7.18, 5.7.19, v5.7.0, v5.7.1, v5.7.10, v5.7.11, v5.7.15, v5.7.2, v5.7.20, v5.7.21, v5.7.22, v5.7.23, v5.7.26, v5.7.27, v5.7.28, v5.7.3, v5.7.4, v5.7.5, v5.7.6, v5.7.7, v5.7.8, v5.7.9, v5.8.0, v5.8.11, v5.8.12, v5.8.14, v5.8.15, v5.8.17, v5.8.18, v5.8.19, v5.8.2, v5.8.20, v5.8.22, v5.8.24, v5.8.27, v5.8.28, v5.8.29, v5.8.3, v5.8.30, v5.8.31, v5.8.32, v5.8.33, v5.8.34, v5.8.35, v5.8.4, v5.8.8, v5.8.9, v6.0.0, v6.0.1, v6.0.2, v6.0.3, v6.0.4].
    - Can only install one of: laravel/framework[v6.1.0, v6.0.4].
    - don't install illuminate/events 5.7.17|don't install laravel/framework v6.1.0
    - don't install illuminate/events 5.7.18|don't install laravel/framework v6.1.0
    - don't install illuminate/events 5.7.19|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.0|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.1|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.10|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.11|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.15|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.2|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.20|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.21|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.22|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.23|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.26|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.27|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.28|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.3|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.4|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.5|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.6|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.7|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.8|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.7.9|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.0|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.11|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.12|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.14|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.15|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.17|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.18|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.19|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.2|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.20|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.22|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.24|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.27|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.28|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.29|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.3|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.30|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.31|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.32|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.33|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.34|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.35|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.4|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.8|don't install laravel/framework v6.1.0
    - don't install illuminate/events v5.8.9|don't install laravel/framework v6.1.0
    - don't install illuminate/events v6.0.0|don't install laravel/framework v6.1.0
    - don't install illuminate/events v6.0.1|don't install laravel/framework v6.1.0
    - don't install illuminate/events v6.0.2|don't install laravel/framework v6.1.0
    - don't install illuminate/events v6.0.3|don't install laravel/framework v6.1.0
    - don't install illuminate/events v6.0.4|don't install laravel/framework v6.1.0
    - don't install illuminate/view 5.7.17|don't install laravel/framework v6.1.0
    - don't install illuminate/view 5.7.18|don't install laravel/framework v6.1.0
    - don't install illuminate/view 5.7.19|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.0|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.1|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.10|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.11|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.15|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.2|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.20|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.21|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.22|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.23|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.26|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.27|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.28|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.3|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.4|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.5|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.6|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.7|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.8|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.7.9|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.0|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.11|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.12|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.14|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.15|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.17|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.18|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.19|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.2|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.20|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.22|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.24|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.27|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.28|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.29|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.3|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.30|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.31|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.32|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.33|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.34|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.35|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.4|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.8|don't install laravel/framework v6.1.0
    - don't install illuminate/view v5.8.9|don't install laravel/framework v6.1.0
    - don't install illuminate/view v6.0.0|don't install laravel/framework v6.1.0
    - don't install illuminate/view v6.0.1|don't install laravel/framework v6.1.0
    - don't install illuminate/view v6.0.2|don't install laravel/framework v6.1.0
    - don't install illuminate/view v6.0.3|don't install laravel/framework v6.1.0
    - don't install illuminate/view v6.0.4|don't install laravel/framework v6.1.0
    - Installation request for laravel/framework ^6.1 -> satisfiable by laravel/framework[v6.1.0].
    - Installation request for glhd/aire ^1.8.0 -> satisfiable by glhd/aire[1.8.0].

```

**Comment by svenewers** [comment]
Just opened PR #41 that solves this by removing the, as far as I'm aware not entriely necessary, version requirements on `laravel/framework` from this project's `composer.json`. In the meantime you could install this package through my github fork to get it to work on Laravel 6.1 and then revert back to the official repo once the issue is solved here as well.

To install from the forked repo add the following to your project's `composer.json`:

```
"repositories": [{
        "type": "package",
        "package": {
            "name": "glhd/aire",
            "version": "master",
            "source": {
                "url": "https://github.com/svenewers/aire.git",
                "type": "git",
                "reference": "master"
            }
        }
    }],
    "require": {
        "glhd/aire": "master",
        ...
    }
```

**Comment by inxilpro** [comment]
This is fixed in [release `1.8.1`](https://github.com/glhd/aire/releases/tag/1.8.1).
